### PR TITLE
Ifttt support

### PR DESCRIPTION
**Pull request by Woodwell**
Motivation: my hue hub link button appears to be broken.   So I came up with these changes to notifiy IFTTT webhooks when video/audio stops and starts.      I'm driving the hue from IFTTT, but really anything could happen from there.

This is my first go progam and my first non-trivial use of git(hub).   Apologies for anything messed up.

### CODE REVIEW

**Comment by patcable** [comment]
This is pretty awesome, thanks so much for your PR! 

I reckon things worked OK? Everything seems pretty reasonable to me as far as the PR goes.

File: ifttt.go [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package main
+
+// OnAir: a golang application that will change the color of a Philips Hue light when
+// your computer's camera or microphone is enabled. You should take a look at the
+// README.md for more information on how to use this application.
+//
+// hue.go: a collection of functions for actually having the Hue change colors or
```
Comment (patcable) [inline ifttt.go]: ah, this is the only thing, and it's quite minor: would just update this header :+1:

**Comment by Woodwell** [comment]
Yep, it works :-).  Fixed the comment.

**Comment by patcable** [comment]
Super appreciate you updating all the things in the PR - not always the case in some other projects :) Cheers, and thanks for contributing!

**Comment by Woodwell** [comment]
Ha.  You're welcome.

**Pull request merged**